### DISCUSSION TITLE
Como reagir quando um projeto sendo integrado ao back-end se torna um Frankstein?

### DISCUSSION THREAD

**Discussion by daniellucindo** [frontendbr/forum#452] [Discussão]
Quando você tem um projeto e no Back - end e quebra tudo, não fecha div no lugar certo ( quebrando o layout ), e o programador esta preocupado com a funcionalidade não com projeto todo...

**Comment by woliveiras** [comment]
Já foi tentado uma reunião sobre isso, @daniellucindo?

Tipo, sentar com o dev que está fazendo as coisas ruins e jogado na mesa pra ele pensar um pouco.

Um almoço seria até melhor do que uma reunião, menos formal e mais troca de ideia livre mesmo.

As vezes a pessoa faz coisa zoada por que tem em mente um prazo de entrega e não um produto que vai crescer a ser desenvolvido com certa atenção.

Em ultimo caso esfrega o Clean Code na cara do peão!!! :joy:

**Comment by leandroruel** [comment]
e em último caso xinga ele pelo twitter. kidding... passa um e-mail para ele informando os problemas causados pelo código backend com prints de tela e exemplos de código. pelo e-mail fica menos chato falar se você não o conhecer muito.

**Comment by felipefialho** [comment]
Cara, precisa de um alinhamento nessa equipe.

A entrega do produto é responsabilidade de todos e para que isso aconteça todos da equipe precisam estar alinhados.
